Source/OCMock/OCClassMockObject.m

```diff
@@ -52,6 +50,20 @@ - (Class)mockedClass
 	return mockedClass;
 }
 
+- (void)assertClassIsSupported:(Class)cls {
+    if(cls == Nil)
+    {
+        [NSException raise:NSInvalidArgumentException format:@"Class cannot be Nil."];
+    }
+    if([cls respondsToSelector:@selector(reasonClassDoesNotSupportMocking)])
```

Comment:
If we're using a formal protocol, should we check for the protocol here (using conformsToProtocol:), rather than implementation of the method?

Reply:
I was checking based on the selector because I didn't want release code to have to be linking in ocmock code.

Reply:
Of course! That makes complete sense.